tools/mason/MasonPublish.chpl

```diff
@@ -267,6 +267,22 @@ proc dryRun(username: string, registryPath : string, isLocal : bool) throws {
     }
   }
   else {
+    const spacer = '------------------------------------------------------';
+    writeln('Checking Registry with ' + registryPath + ' path.');
+    var registryTest = registryPathCheck(registryPath, username, false);
+    writeln(spacer);
+    writeln('The current mason environment is:');
+    returnMasonEnv();
+    var regExists: bool = false;
+    const regUrl = "https://github.com/chapel-lang/mason-registry";
```

Comment:
The default registry is defined in `MasonEnv.MASON_REGISTRY`. You could promote that to `MasonEnv` module-level const, and then use that directly to reduce the number of places we have this hard-coded URL in the source code.